tests/models/virtualfolder.py

```diff
@@ -314,3 +286,17 @@ def test_vfolder_calc_priority(settings, store0):
     assert store0.calculate_priority() == 5.0
     settings.INSTALLED_APPS.remove("virtualfolder")
     assert store0.calculate_priority() == 1.0
+
+
+@pytest.mark.django_db
+def test_vfolder_stats(settings, tp0, store0):
+    tp0_stores = ",".join(
+        p[len(tp0.pootle_path):]
+        for p in tp0.stores.values_list("pootle_path", flat=True))
+    vf0 = VirtualFolder.objects.create(
+        name="vf0",
+        title="the vf0",
+        location=tp0.pootle_path,
+        filter_rules=tp0_stores)
+    for vfti in vf0.vf_treeitems.iterator():
+        pass
```

Comment:
Also test that stats are correctly updated after:

- Changing the units in some store (obsoleting, resurrecting, removing, adding)
- Changing the stores in the vfolder (removing, adding)